### DISCUSSION TITLE
Column sorting in Tables

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1700] [Q&A] [answered]
Hello, 

Are there plans to add column sorting to Paste tables in the near future. If not, is there a way we could achieve that today if we're using Paste tables?

Thank you!

**Comment by TheSisb** [comment, marked as answer]
Hey @PoojMenon,

We have plans to improve our Tables in the future and document sorting and filtering patterns, but no timeline yet. In the meantime, one of the designers (Sang) came up with this really neat solution. Hope this helps!


![image](https://user-images.githubusercontent.com/1592327/127235156-b116a867-f3f0-4e09-b052-76a9e2dc3ac8.png)
